modules/docs/lib/ExampleCodeBlock.tsx

```diff
@@ -70,31 +82,55 @@ export const ExampleCodeBlock = ({code}: any) => {
     navigator.clipboard.writeText(textInput.current.textContent);
   };
 
-  // React.useEffect(() => {
-  //   if (copied) {
-  //     const copyCodeTimeout = setTimeout(() => {
-  //       setCopied(false);
-  //     }, 2000);
```

Comment:
Do we want to remove this? Or keep it commented.

Reply:
if it's not being used, I just removed it.